benchkit/commandattachments/tracecmd.py

```diff
@@ -0,0 +1,41 @@
+from benchkit.shell.shellasync import AsyncProcess
+from benchkit.utils.types import PathType
+from benchkit.platforms import get_current_platform
+import pathlib
+
+class trace_cmd_attachement():
+
+    def __init__(
+        self,
+        events: list[str] = []
+    ):
+        self.events = events
+
+    def attachement(
+                self,
+                process: AsyncProcess,
+                record_data_dir: PathType,
+        ) -> None:
+            platform = get_current_platform()
```

Comment:
take the platform as an argument and if it is `None` you call `get_current_platform()`